manifests/kustomize/env/namespaced-installation/kustomization.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+- ../platform-agnostic/minio
+- ../platform-agnostic/mysql
+- ../../base/argo
+- ../../base/pipeline
+- ../../base/metadata
+
+resources:
+- namespace.yaml
+
+namespace: kubeflow
+
+images:
+- name: mysql
+  newTag: "5.6"
+- name: minio/minio
+  newTag: RELEASE.2018-02-09T22-40-05Z
+
+configMapGenerator:
+- name: pipeline-install-config
+  env: params.env
+
+secretGenerator:
+- name: mysql-secret
+  env: params-db-secret.env
+
+vars:
```

Comment:
I hope we can avoid this duplication, but haven't thought of a good idea. We can go with this.

Reply:
The major reason we have duplication here is because `base` has everything included. Do you think we can remove optional packages like `cache` and `applications` from base and then add them into `standalone` which is `bases:base` + `resources: cache` + `resources:application?  

In this case, all existing manifest can change to use `standalone` and this manifest can use `base` which is pretty clean.

Reply:
That's a great idea! Do you want to directly refactor in this PR?